source/cfp.rst

```diff
@@ -0,0 +1,107 @@
+
+About you
+---------
+
+We'd like to know a little bit about you for our files. If you submit more than
+one proposal, you don't need to send all of these details again.
+
+Your full name
+  --
+Your email address
+  --
+More information about you (public)
+  Provide us with some information that we can publish on the website or in the
+  programme. A paragraph is enough, but feel free to write more.
+More information about you (private)
+  Add anything else you feel we should know about you. This will be kept
+  private.
+I'm happy to be recorded and/or broadcast on the Internet
+  yes/no
+Title
+  --
+What are you proposing?  (One of)
+  * A talk
+  * A workshow or tutorial
+  * A code clinic
+  * Poster
+  * Something completely different
+Abstract (public)
+  Describe what you'll be talking about or doing.
+Why do you think people will be interested in this? (private)
+  --
+Tell us about any requirements your session has
+  WiFi and video projector will be provided for talks; will you need anything else?
+What level of audience technical experience are you aiming at?  (Any of)
+  * Absolute beginner
+  * Novice
+  * Moderately experienced
+  * Highly experienced
+  * Not applicable
+  * Other
+
+
+Talk proposal
+-------------
+
+Please note that the final running time of your talk will be agreed with the
+conference programme organisers.
+
+Talk format (One of)
+  * Shorter talk (20 minutes including questions)
+  * Longer talk (45 minutes including questions)
+  * Longer preferred, but can make it shorter
+
+
+Workshop/tutorial proposal
+--------------------------
+
+Please note that the final logistical details of your session will be agreed
+with the conference programme organisers.
+
+Timing
+  How long should the workshop run? Is it one that you'd like to do more than
+  once if appropriate?
+Maximum number of people you can manage
+  --
+Would you like helpers?
+  How many do you need, and what skills will they require?
+
+
+Code clinic
+-----------
+
+Please note that the final logistical details of your session will be agreed
+with the conference programme organisers.
+
+Timing
+  How long would you like to run your clinic? Is it one that you'd like to do
+  more than once if appropriate?
+Maximum number of people you can manage
+ --
+Would you like helpers?
+  How many do you need, and what skills will they require?
+
+
+Something completely different
+------------------------------
+
+A piece of Python performance art? A daring experiment? A mass-participation
+event? Something even we haven't imagined? Tell us about it.
+
+I'm proposing...
+  Give us as much information about this as you can.
+
+
+After submission
+----------------
+
+Thanks for sending your proposal. Please take a note of the link that below, so
+you can edit your proposal later. Note also that Google forms don't send
+confirmation by email, but if you can see this message, it means we've received
+your proposal.
```

Comment:
It looks like it's possible to send people an email response: http://stackoverflow.com/questions/10710354/how-to-add-edit-response-link-to-google-forms-emails.